### PR TITLE
implemented installroot option

### PR DESCRIPTION
**Pull request by Alexander198961**
SUMMARY

added installroot option to zypper module
ISSUE TYPE

    Feature Pull Request

COMPONENT NAME

module
ANSIBLE VERSION

ansible 2.4.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = Default w/o overrides
  python version = 2.7.9 (default, Dec 21 2014, 11:02:59) [GCC

ADDITIONAL INFORMATION

This is fix for  https://github.com/ansible/ansible/pull/28651 (See this tread)

### CODE REVIEW

**Review by gundalow — APPROVED** [review]

File: lib/ansible/modules/packaging/os/zypper.py [hunk 1]
```diff
@@ -62,6 +62,12 @@
         choices: [ package, patch, pattern, product, srcpackage, application ]
         default: "package"
         version_added: "2.0"
+    extra_args_precommand:
+       version_added: "2.5"
+       required: false
+       description:
+         - Add additional  global target options to C(zypper) .
```
Comment (robinro) [inline lib/ansible/modules/packaging/os/zypper.py]: double space and space before dot.

**Review by robinro — APPROVED** [review]
Looks good overall. Thanks @Alexander198961.

File: lib/ansible/modules/packaging/os/zypper.py (hunk 1 above)
Comment (Alexander198961) [reply lib/ansible/modules/packaging/os/zypper.py]: fixed

**Comment by robinro** [comment]
Thanks!
shipit

**Comment by Alexander198961** [comment]
@gundalow I have resolved rebase issue can this commit will be  merged ?

**Comment by Alexander198961** [comment]
@gundalow  can this will be merged ?

**Review by gundalow — APPROVED** [review]

**Comment by gundalow** [comment]
Merged into `devel` for release in Ansible 2.6.0.

Thanks!

**Pull request merged**